health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/README.md

```
@@ -6,18 +1,84 @@
+
+- [ramkrishna-egov](https://github.com/ramkrishna-egov)
+- [rachna-egov](https://github.com/rachna-egov)
+- [pitabash-eGov](https://github.com/pitabash-eGov)
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
+
+## Documentation
+
+Documentation Site (https://core.digit.org/guides/developer-guide/ui-developer-guide/digit-ui)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)